Core.Collectors/Web/HttpExceptionSignature.cs

```diff
@@ -1,29 +1,49 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using Microsoft.CloudMine.Core.Collectors.Collector;
 using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace Microsoft.CloudMine.Core.Collectors.Web
-{
-    public class HttpExceptionSignature
+{ 
+    public class HttpExceptionSignature : IAllowListStatus
     {
-        public static readonly HttpExceptionSignature RequestTimeoutException = new HttpExceptionSignature(exception => 
+        public static HttpExceptionSignature RequestTimeoutException(Func<List<CollectionNode>> continuation = null)
```

Comment:
>(Func<List<CollectionNode>> continuation = null) [](http://example.com/codeflow?start=68&length=48)

It looks like that making this a method (instead of a property) is causing quite a bit of changes. Can you do this in a separate PR or just skip doing this?